### ISSUE TITLE
Question about BYOL results

### ISSUE THREAD

**Issue by fjsaezm** [deepmind/deepmind-research#276]
Hello there!

I have been running some experiments with the [BYOL](https://github.com/deepmind/deepmind-research/tree/master/byol) framework. I used the fast iteration with imagenette since I was using a single GPU for the experiments.

Using `batch_size=256` and `ResNet50` , I ran the pretrain mode in the script and *apparently*, It achieved $100\%$ top 1 accuracy score.

- Is this normal?
- Where is this accuracy being computed in ImageNette, since it only has train and validation sets (no test set) ?

Thank u guys!
